galley/pkg/config/resource/name.go

```diff
@@ -52,17 +52,32 @@ func NewFullName(name string) (Name, error) {
 	return Name{string: name}, nil
 }
 
+// NewShortOrFullName tries to parse the given name to resource.Name. If the name does not include namespace information,
+// the defaultNamespace is used.
+func NewShortOrFullName(name, defaultNamespace string) Name {
```

Comment:
Is there some sort of logic inside Istio for the order of name, namespace? the original reason for this order (instead of the same order) was to emphasize the default parameter as something that's not necessarily used.

Reply:
AFAICT the ordering is just for convenience and readability, given that namespace/name is the order it appears in in string form.